src/genome.c

```diff
@@ -1,52 +1,133 @@
 /*
-	Smart search for upgrade paths
-	Copyright (C) 2016 Andrew Leeming <andrew.leeming@codethink.co.uk> and
-					   Bob Mottram <bob.mottram@codethink.co.uk>
-
-	This program is free software: you can redistribute it and/or modify
-	it under the terms of the GNU General Public License as published by
-	the Free Software Foundation, either version 3 of the License, or
-	(at your option) any later version.
-
-	This program is distributed in the hope that it will be useful,
-	but WITHOUT ANY WARRANTY; without even the implied warranty of
-	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-	GNU General Public License for more details.
-
-	You should have received a copy of the GNU General Public License
-	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  Smart search for upgrade paths
+  Copyright (C) 2016 Andrew Leeming <andrew.leeming@codethink.co.uk> and
+  Bob Mottram <bob.mottram@codethink.co.uk>
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #include "scalam.h"
 
 /**
- * @brief Mutates a given genome
+ * @brief returns mutation rate expressed as an integer value
+ * @param population The population in which the genome exists
+ * @returns mutation rate value in the range 0 -> 1000
+ */
+int genome_mutability(sc_population * population)
+{
+	return (int)(population->mutation_rate * 1000);
```

Comment:
There are multiple occurrences of the 1000 scalar. #define SC_MUTATION_SCALAR 1000. instead?